app/templates/eventView.html

```diff
@@ -123,5 +123,12 @@ <h3><br>Program Trainings</h3>
                     <button type="submit" class="btn btn-primary" value="{{eventData.id}}" id="rsvpBtn">RSVP</button>
                  {% endif %}
               {% endif %}
+              {% if g.current_user.isCeltsAdmin or isProgramManager %}
+            <div class="d-flex align-items-center">
+                <i class="bi bi-eye-fill me-2"></i>
+                <span class="me-2">Views:</span>
+                <span id="view-count">{{viewCount}}</span>
```

Comment:
We have been using camelCase for id names everywhere else, please keep it contestant.